src/core/model_config_utils.h

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2018-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2018-2021, NVIDIA CORPORATION & AFFILIATES. All rights
```

Comment:
Remove `(c)` to avoid copyright wrap.

Reply:
This change came from the formatter I think